snowfakery/output_streams.py

```diff
@@ -68,7 +68,7 @@ def flatten(
         sourcetable: str,
         fieldname: str,
         source_row_dict: Dict,
-        target_object_row: ObjectRow,
+        target_object_row: [ObjectRow, NicknameSlot],
```

Comment:
I'm not familiar with this syntax in type annotations. Does it actually work, or is it supposed to be `Union[ObjectRow, NicknameSlot]`? There are a few repetitions below too.

Reply:
That was a typo and for some reason TypeGuard didn't catch it. Fixed now.

Reply:
https://github.com/SFDO-Tooling/Snowfakery/commit/29e017095a45b903743fab85bf2ea53b174c8040#diff-bd3afc575a9fee6d617119045da1b50eR71